File: 5.1차원배열/2920번 음계.cpp
#include <iostream>
using namespace std;
int main(){
	int n=0;
	int p=10000000;
	int temp;
	for (int i = 0; i < 8; i++)
	{
		cin >> temp;
		n+=p*temp;
		p=p/10;
	}

	if (n==12345678) cout << "ascending";
	else if(n==87654321) cout << "descending";
	else cout << "mixed";
}

File: java 로 풀어본 문제/2920번 음계.java
import java.io.IOException; 
import java.util.Scanner;
public class Main { 
	public static void main(String[] args) throws IOException {

	Scanner sc=new Scanner(System.in);
		boolean Ascencding=true;
		boolean Decending=true;
		int[] input = new int[8];
		for(int i=0; i<8; i++){
			input[i]=sc.nextInt();
		}
		for (int i=1;i<8;i++) {
			if (input[i]>input[i-1]){
				Decending=false;
			}
			else if(input[i]<input[i-1]){
				Ascencding=false;
			}
		}

		if (Decending){
			System.out.print("descending");
		}
		else if(Ascencding){
			System.out.print("ascending");
		}
		else{
			System.out.print("mixed");
		}
	} 
}

